src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -10676,7 +10676,13 @@ private ExpressionSyntax ParseArrayOrObjectCreationExpression()
                         SyntaxFactory.MissingToken(SyntaxKind.CloseParenToken));
                 }
 
-                return _syntaxFactory.ObjectCreationExpression(@new, type, argumentList, initializer);
+                var objectCreation = _syntaxFactory.ObjectCreationExpression(@new, type, argumentList, initializer);
+                if (type.Kind == SyntaxKind.TupleType)
+                {
+                    objectCreation = this.AddError(objectCreation, objectCreation, ErrorCode.ERR_NewWithTupleTypeSyntax, type.ToString());
```

Comment:
Error gets truncated. It does look ugly. 
I will put squigglies only for the type and not mention the actual type i the error

---
In reply to: [81658408](https://github.com/dotnet/roslyn/pull/14161#discussion_r81658408) [](ancestors = 81658408,81653476)